core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -257,8 +257,9 @@ class LogCleaner(val config: CleanerConfig,
             cleanerManager.doneDeleting(topicPartition)
           }
       }
-      if (!cleaned)
-        backOffWaitLatch.await(config.backOffMs, TimeUnit.MILLISECONDS)
+      if (!cleaned) {
+        val _ = backOffWaitLatch.await(config.backOffMs, TimeUnit.MILLISECONDS)
```

Comment:
The two-argument form of `CountDownLatch#await` returns `true` if the latch was decremented, and `false` if it timed out.  This code doesn't check this return value.  It's not a bug, because there is code later on which is calling `checkDone` to determine if the cleaner should exit.  So we don't need the return value from `await`.  But it triggers an "ignored return value" findbugs warning.  Using `val _ = ...await...` suppresses this.